### PR TITLE
feat: tidy the generated go tag

### PR DESCRIPTION
**Pull request by FGYFFFF**
#### What type of PR is this?

feat
#### What this PR does / why we need it (English/Chinese):

en: Change the generated go tag policy to remove redundant go tags
zh: 改变生成的go tag 策略，删除掉冗余的go tag

### CODE REVIEW

**Comment by FGYFFFF** [comment]
删除掉冗余的tag，thrift和protobuf采用同样的策略。
等待 thriftgo no_json_tag 特性合入&发版中 [51](https://github.com/cloudwego/thriftgo/pull/51)

File: cmd/hz/internal/thrift/plugin.go [hunk 1]
```diff
@@ -254,13 +255,6 @@ func (plugin *Plugin) InsertTag() ([]*thriftgo_plugin.Generated, error) {
 			stName := st.GetName()
 			for _, f := range st.Fields {
 				fieldName := f.GetName()
-				hasBodyTag := false
```
Comment (FGYFFFF) [inline cmd/hz/internal/thrift/plugin.go:257]: 之前thrifrgo自带"json"tag，这里是为了配合thriftgo；现在不用了

File: cmd/hz/internal/util/tool_install.go [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+}
+
+// CheckAndUpdateThriftgo checks the version of thriftgo and updates the tool to the latest version if its version is less than v0.2.0.
+func CheckAndUpdateThriftgo() error {
+	path, err := exec.LookPath(meta.TpCompilerThrift)
+	if err != nil {
+		return fmt.Errorf("can not find %s", meta.TpCompilerThrift)
+	}
+	curVersion, err := QueryVersion(path)
+	logs.Infof("%s", curVersion)
```
Comment (Duslia) [inline cmd/hz/internal/util/tool_install.go:141]: 日志加一个说明吧，比如   current version: %s, xxxx

Comment (FGYFFFF) [reply cmd/hz/internal/util/tool_install.go]: done

**Review by welkeyever — APPROVED** [review]

**Review by Duslia — APPROVED** [review]

**Pull request merged**